installer/upgrader/console.php

```diff
@@ -21,11 +21,12 @@
 use OrangeHRM\Framework\Services;
 use OrangeHRM\ORM\Doctrine;
 use OrangeHRM\Tools\Migrations\Version20220125;
+use OrangeHRM\Tools\Migrations\Version20220221;
 
 require_once realpath(__DIR__ . '/../../symfony/vendor/autoload.php');
 
 ServiceContainer::getContainer()->register(Services::DOCTRINE)
     ->setFactory([Doctrine::class, 'getEntityManager']);
 
-$migration = new Version20220125();
+$migration = new Version20220221();
```

Comment:
you can't change this, cz without `Version20220125` migration application won't work

Reply:
for now I've created a new php file to handle the Version20220221 called translateHandler.php this could be run independently from the console